tests/testdataset.py

```diff
@@ -480,3 +480,35 @@ def test_dataset_sort_vdim_hm(self):
     def test_dataset_groupby(self):
         self.assertEqual(self.dataset_hm.groupby('x').keys(), list(self.xs))
 
+
+
+class IrisDatasetTest(GridDatasetTest):
+    """
+    Tests for Iris interface
+    """
+
+    def setUp(self):
+        import iris
+        self.restore_datatype = Dataset.datatype
+        Dataset.datatype = ['cube']
+        self.data_instance_type = iris.cube.Cube
+        self.init_data()
+
+    def test_dataset_add_dimensions_values_hm(self):
```

Comment:
Yes, sounds good. They should eventually be remove once those methods are implemented.